avd_docs/kubernetes/general/AVD-KSV-0041/docs.md

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
So I see this was added (generated) but the corresponding RBAC docs are still present: https://github.com/sm171190/defsec/blob/sa-migrating-rbac-to-kubernetes-scanner/avd_docs/rbac/general/AVD-KSV-0041/docs.md


I assume there's no point in keeping RBAC docs, correct?

Reply:
That's correct. We'll have to remove the RBAC docs.